resources/lang/en/patrols/new_cat.json

```
@@ -4327,5 +4327,393 @@
+                        "amount": 10
+                    }
+                ],
+                "outsider_rep": 1
+            },
+            {
+                "text": "A loner? Interesting. Never one to pass up a good conversation, s_c saunters up to the outsider and begins chatting with {PRONOUN/n_c:0/object} as though they're old friends. And by the time the two young cats part ways, it feels like they are.",
+                "exp": 10,
+                "weight": 40,
+                "stat_trait": ["charismatic", "confident", "playful"],
+                "new_cat": [
+                    ["meeting", "loner", "age:adolescent"]
+                ],
+                "relationships": [
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["patrol"],
+                        "mutual": true,
+                        "values": ["platonic", "trust", "comfort"],
+                        "amount": 15
+                    }
+                ],
+                "outsider_rep": 1
+            },
+            {
+                "text": "Oh wow, a kittypet! Sure, <i>technically</i> s_c should be sending the outsider away, but {PRONOUN/s_c/subject} just {VERB/s_c/have/has} so many questions! Luckily, the kittypet is willing to sate s_c's curiosity, and the Clancat eventually returns to camp with a new understanding, and potentially a new friend.",
+                "exp": 10,
+                "weight": 20,
+                "stat_trait": ["charismatic", "confident", "playful"],
+                "new_cat": [
+                    ["meeting", "kittypet", "age:adolescent"]
+                ],
+                "relationships": [
+                    {
+                        "cats_from": ["n_c:0"],
+                        "cats_to": ["patrol"],
+                        "mutual": true,
+                        "values": ["platonic", "trust", "comfort"],
+                        "amount": 15
+                    }
+                ],
+                "outsider_rep": 1
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "A loner? How interesting. p_l approaches the outsider but, before {PRONOUN/p_l/subject} can even meow out a greeting, the loner takes off, evidently startled by the c_n cat's appearance.",
+                "exp": 0,
+                "weight": 20,
+                "new_cat": [
+                    ["meeting", "loner", "age:adolescent"]
+                ],
+                "outsider_rep": 0
+            },
+            {
+                "text": "What in StarClan's name is a <i>kittypet</i> doing here? c_n's territory is no place for such a pampered creature, so p_l approaches the other young cat, intending to send {PRONOUN/n_c:0/object} back to wherever {PRONOUN/n_c:0/subject} came from. However, p_l's presence clearly intimidates the kittypet, who bolts away the moment {PRONOUN/n_c:0/subject} {VERB/n_c:0/see/sees} {PRONOUN/p_l/object}.",
```

```suggestion
                "text": "What in StarClan's name is a <i>kittypet</i> doing here? c_n's territory is no place for such a pampered creature. p_l approaches the other young cat, intending to send {PRONOUN/n_c:0/object} back to wherever {PRONOUN/n_c:0/subject} came from. However, p_l's presence clearly intimidates the kittypet, who bolts away the moment {PRONOUN/n_c:0/subject} {VERB/n_c:0/see/sees} {PRONOUN/p_l/object}.",
```

I'm not sure "however" is the right word here. It implies that the sentence is opposite to the goals or purposes of the previous, but that's not true in this case: p_l wanted to scare the cat off the territory, and it worked. If the implication is trying to be "oh God, a kittypet, it'll die out here, I need to help the poor thing back to things it understands", then maybe swapping "send" to "help" or "guide"?

---

Edited so that "send" has become "guide"!

---

Also need to fix the run-on sentence here! Make sure you push that change to GitHub because it's not showing up here yet.